index.html

```diff
@@ -185,142 +185,527 @@ <h5 class="sr-only">숙소 정보</h5>
+                  media="(min-width: 1127.1px) and (max-width: 1439px)" />
+                <source srcset="https://a0.muscache.com/im/pictures/78fb8268-b999-4389-b072-e2a66910e41b.jpg?im_w=1680 1x"
+                  media="(min-width: 1439.1px)" />
+                <img src="https://a0.muscache.com/im/pictures/78fb8268-b999-4389-b072-e2a66910e41b.jpg?im_w=720" alt="푸른 바다 가운데 섬이 있다">
+              </picture>
             </div>
             <h5 class="sr-only">숙소 정보</h5>
             <!-- 숙소 정보 변경 -->
             <ul class="accommodation__item__brief">
-              <li class="font-bold"><abbr title="Maldiv">MV, </abbr>몰디브</li>
-              <li>6,596km 거리</li>
-              <li>9월 24일~29일</li>
-              <li class="font-bold">₩781,552 /박</li>
+              <li class="font-bold">El Nido, 필리핀</li>
+              <li>3,050km 거리</li>
+              <li>8월 11일~18일</li>
+              <li class="font-bold">₩837,686 /박</li>
+              <li
+                class="review"
+                aria-label="평점"
+              >
+                4.96
+              </li>
+            </ul>
+          </a>
+        </article>
+        <article class="accommodation__item">
+          <a class="accommodation__link" href="/">
+            <!-- h4 변경 -->
+            <h4 class="sr-only">몰디브에 위치한 비치 풀 빌라</h4>
+            <div class="accommodation__item__image">
+              <!-- src, alt 변경 -->
+              <picture>
+                <source
+                  srcset="https://a0.muscache.com/im/pictures/miso/Hosting-52159627/original/d727cf6e-4787-4b38-a86f-0cd8d91d02b2.png?im_w=320 1x"
+                  media="(max-width: 743px)" />
+                <source
+                  srcset="https://a0.muscache.com/im/pictures/miso/Hosting-52159627/original/d727cf6e-4787-4b38-a86f-0cd8d91d02b2.png?im_w=720 1x"
+                  media="(min-width: 743.1px) and (max-width: 1127px)" />
+                <source
+                  srcset="https://a0.muscache.com/im/pictures/miso/Hosting-52159627/original/d727cf6e-4787-4b38-a86f-0cd8d91d02b2.png?im_w=1200 1x"
+                  media="(min-width: 1127.1px) and (max-width: 1439px)" />
+                <source
+                  srcset="https://a0.muscache.com/im/pictures/miso/Hosting-52159627/original/d727cf6e-4787-4b38-a86f-0cd8d91d02b2.png?im_w=1680 1x"
+                  media="(min-width: 1439.1px)" />
+                <img
+                  src="https://a0.muscache.com/im/pictures/miso/Hosting-52159627/original/d727cf6e-4787-4b38-a86f-0cd8d91d02b2.png?im_w=720"
+                  alt="바다 위에 나무로 된 빌라들이 섬까지 줄지어 서있다" />
+              </picture>
+            </div>
+            <h5 class="sr-only">숙소 정보</h5>
+            <!-- 숙소 정보 변경 -->
+            <ul class="accommodation__item__brief">
+              <li class="font-bold">Atoll, 몰디브</li>
+              <li>6,519km 거리</li>
+              <li>8월 10일~16일</li>
```

Comment:
이런 식으로 하면 더 좋을 것 같아요!
```suggestion
              <li>8월 10일<span aria-label="부터">~<span>16일</li>
```